.github/workflows/cronjobs.yml

```diff
@@ -5,7 +5,7 @@ name: Cronjobs
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '* * * * *' # every minutes
+    - cron:  '0 * * * *' # every hours
```

Comment:
What is this dummy job for?

Reply:
Test for cronjob now

Reply:
maybe don't merge into the master branch if it's only a test job, better move to a personal repo or a forked repo.

Reply:
I had discussed this with @Yancey1989 offline. I agree that dummy cronjob is useless for BladeDISC. But it helps us to have a quick test on the cronjob workflow for agile development purposes. We can delete the dummy cronjob after the benchmark cronjob is set up.

Reply:
OK, I run benckmark in other branch for now.